.github/workflows/build-latest-image.yml

```diff
@@ -9,52 +9,22 @@ on:
 env:
   TERM: xterm-color
   REGISTRY: quay.io
-jobs:
-  shellcheck:
-    name: Shellcheck
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v4
-      - name: Run ShellCheck
-        uses: ludeeus/action-shellcheck@master
-        env:
-          SHELLCHECK_OPTS: -e SC2068
-        with:
-          scandir: './scripts'
-          severity: error
-  
-  yamllint:
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v4
-      - name: yaml-lint
-        uses: ibiqlik/action-yamllint@v3
-        with:
-          config_data: |
-            extends: default
-            rules:
-              line-length:
-                level: warning
-              trailing-spaces:
-                level: warning
-              brackets:
-                level: warning
-              empty-lines:
-                level: warning
-
-  
+jobs:  
   build-exe:
     name: Build and push image
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     env:
       SHELL: /bin/bash
       KUBECONFIG: '/home/runner/.kube/config'
 
     steps:
-      - name: Set up Go 1.21.4
+      - name: Set up go 1.22
```

Comment:
Latest version of Go is 1.24

https://github.com/redhat-best-practices-for-k8s/certsuite/pull/2899